lib/in_out/saver.py

```diff
@@ -72,15 +72,18 @@ def save_csv(matrix, output_directory, output_file_name):
         Saves the input matrix as a CSV File
 
     Args:
-        nx_graph(str): networkx graph object to be drawn
+        matrix(list):  an array containing data to be saved
         output_drectory(str): location to save graph
-        output_file_name(str): name of the image file to be saved
+        output_file_name(str): name of the csv file to be saved
 
     Returns:
        null
 
     """
     print "Generating", output_file_name + ".csv"
     check_if_dir_exists(output_directory) #create output directory if doesn't exist
-    np.savetxt(output_directory + "/" + output_file_name +".csv", matrix, delimiter=",")
-    
+    file = output_directory + "/" + output_file_name +".csv"
```

Comment:
Since i cannot attach csv files here i am attaching the exponential fit graph
new
![conv_len](https://cloud.githubusercontent.com/assets/22174615/23330073/6db53d8c-fb6b-11e6-92d5-c0bc7b720267.png)
old
![conv_len](https://cloud.githubusercontent.com/assets/22174615/23330088/bf6e4a60-fb6b-11e6-9cf1-715422d76499.png)
The csv files generated are also identical.